### PR TITLE
Allow configuring results to collapse as parallel children on overview

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/111470#note-5

@AdamWill

### CODE REVIEW

File: templates/webapi/test/overview.html.ep [hunk 1]
```diff
@@ -5,6 +5,7 @@
 % use List::Util qw(sum);
 
 % content_for 'ready_function' => begin
+  window.overviewOkResultSelector = '<%= $ok_result_sel %>';
```
Comment (okurz) [inline templates/webapi/test/overview.html.ep:8]: this looks to be inconsistent. In other places you say "…collapsable_results_sel" but here "ok_result_sel" even though it's configurable so not necessarily the same as the real definition of "ok" results that we use elsewhere

**Review by okurz — CHANGES_REQUESTED** [review]

File: templates/webapi/test/overview.html.ep (hunk 1 above)
Comment (Martchus) [reply templates/webapi/test/overview.html.ep:8]: My thinking was the selector defines what "ok" results are in that particular occasion. However, it is likely better to stick with "collapsable_results_sel".

**Comment by AdamWill** [comment]
Well, it's definitely nice for me to have an option to go back to the old way. The concern is always that adding configurable features adds complexity and can cause less-used paths to break, but in this case it doesn't look too bad.

**Comment by Martchus** [comment]
It is actually not a separate code path. All new lines of code are always executed, just with a different number of array elements.

**Comment by AdamWill** [comment]
Yup, that's why I said in this case it looks good :) I meant that was always the *general* concern when adding configurable options.

**Review by okurz — APPROVED** [review]

**Pull request merged**